### PR TITLE
fix: add caip-27 + fix strings in yaml for multichain

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

File: multichain/openrpc.yaml [hunk 1]
```diff
@@ -5,179 +5,223 @@ info:
...
+        description: an object containing a JSON-RPC request with `method` and `params`
+        deprecated: false
+        required: true
         schema:
           type: "object"
+          properties:
+            method:
+              type: "string"
+            params:
+              true
```
Comment (jiexi) [inline multichain/openrpc.yaml:26]: just making sure this is the correct way to specify optional params

**Review by jiexi — APPROVED** [review]

File: multichain/openrpc.yaml (hunk 1 above)
Comment (shanejonas) [reply multichain/openrpc.yaml:26]: its not optional but `true` here would just mean `any` if it was typescript

File: multichain/openrpc.yaml [hunk 2]
```diff
@@ -5,179 +5,223 @@ info:
...
+      - name: provider_requestExample
+        params:
+          - name: scope
+            value: eip155:1
+          - name: request
+            value:
+              method: eth_getBalance
+              params: []
+          - name: sessionId
+            value: "0xdeadbeef"
```
Comment (adonesky1) [inline multichain/openrpc.yaml:46]: is it only `value`s that should remain strings?

**Review by adonesky1 — APPROVED** [review]

File: multichain/openrpc.yaml (hunk 2 above)
Comment (shanejonas) [reply multichain/openrpc.yaml:46]: if i did it with this:
`0xdeadbeef` without the string wrapping then yaml would think its a hex number

Comment (shanejonas) [reply multichain/openrpc.yaml:46]: image

**Pull request merged**